CPackLists.txt

```diff
@@ -18,16 +18,18 @@
  #           COMPONENT headers)
 
 
+   SET(CMAKE_INSTALL_LIBDIR lib CACHE PATH "Output dir for libraries")
+   SET(CMAKE_INSTALL_HEADERDIR include CACHE PATH "Output dir for headers")
 
-   INSTALL( TARGETS g3logger_shared
-            ARCHIVE
-            LIBRARY DESTINATION /usr/local/lib
+   INSTALL( TARGETS g3logger g3logger_shared
+            ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+            LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
             COMPONENT libraries)
 
    INSTALL( FILES ${HEADER_FILES}
-            DESTINATION /usr/local/include/g3log
+            DESTINATION ${CMAKE_INSTALL_HEADERDIR}/g3log
```

Comment:
1. Please put a printout somewhere in the CPackagelist that tells the user what will be installed and where
2. Please describe to the user how to set the locations for library files and header files. I.e. something like

```
To install headers in in <locationX>/g3log and to install library files in <locationY> please use:
cmake -DCMAKE_INSTALL_HEADERDIR=<locationX> -DCMAKE_INSTALL_LIBDIR=<locationY> ..
```